armi/nuclearDataIO/tests/library-file-generation/mc2v3-AB.inp

```diff
@@ -8,8 +8,8 @@ $control
     l_gamma                 = T 
 /
 $library
-     c_mcclibdir  ="\\albert\apps\prod\mc2\3.2.2\libraries\endfb-vii.0\lib.mcc.e70"
-     c_gammalibdir = "\\albert\apps\prod\mc2\3.2.2\libraries\endfb-vii.0\lib.gamma.e70"
+     c_mcclibdir  ="\\path\to\mc2\3.2.2\libraries\endfb-vii.0\lib.mcc.e70"
+     c_gammalibdir = "\\path\to\mc2\3.2.2\libraries\endfb-vii.0\lib.gamma.e70"
```

Comment:
Oh interesting... I don't know what these inputs are needed for in the testing. They should probably not be in the framework.

Reply:
Well, as you can see, these tests all run on my laptop with complete fake "\\path\to\" address.

I'm afraid I can't find documentation on whatever these ".inp" files are, so I don't know if we can just remove these lines entirely, and have the input files still be valid.

Do you know what these files are? Can these lines just be deleted?